fee_components.go

```diff
@@ -0,0 +1,106 @@
+package hedera
+
+import (
+	"fmt"
+	protobuf "github.com/golang/protobuf/proto"
+	"github.com/hashgraph/hedera-sdk-go/v2/proto"
+)
+
+type FeeComponents struct {
+	Min int64
+	Max int64
+	Constant int64
+	TransactionBandwidthByte int64
+	TransactionVerification int64
+	TransactionRamByteHour int64
+	TransactionStorageByteHour int64
+	ContractTransactionGas int64
+	TransferVolumeHbar int64
+	ResponseMemoryByte int64
+	ResponseDiscByte int64
+}
+
+func newFeeComponents(min int64, max int64, constant int64, transactionBandwidthByte int64, transactionVerification int64,
```

Comment:
I don't think there is a point for this to exist really. All the variables here are `int64` and the default value is `0`. Might as well construct the type using the struct initialization notation.